v0/destinations/af/transform.js

```diff
@@ -89,7 +89,7 @@ function responseBuilderSimple(payload, message, destination) {
     updatedPayload.bundleIdentifier = bundleIdentifier;
   }
 
-  const sharingFilter = destination.Config.sharingFilter || "all";
+  const { sharingFilter } = destination.Config;
   if (isDefinedAndNotNull(sharingFilter)) {
```

Comment:
```suggestion
  if (isDefinedAndNotNullAndNotEmpty(sharingFilter)) {
```
We should not send sharingFilter in the payload if it is empty `''`.